src/module.c

```diff
@@ -5660,16 +5683,52 @@ RedisModuleCallReply *RM_Call(RedisModuleCtx *ctx, const char *cmdname, const ch
     cmd = lookupCommand(c->argv,c->argc);
     if (!cmd) {
         errno = ENOENT;
+        if (error_as_call_replies) {
+            sds msg = sdscatfmt(sdsempty(),"ERR Unknown Redis "
+                                           "command %S.",c->argv[0]->ptr);
+            reply = callReplyCreateError(msg, ctx);
+        }
         goto cleanup;
     }
     c->cmd = c->lastcmd = c->realcmd = cmd;
 
     /* Basic arity checks. */
     if ((cmd->arity > 0 && cmd->arity != argc) || (argc < -cmd->arity)) {
         errno = EINVAL;
+        if (error_as_call_replies) {
+            sds msg = sdscatfmt(sdsempty(), "ERR Wrong number of "
+                                            "args calling Redis command %S.", c->argv[0]->ptr);
+            reply = callReplyCreateError(msg, ctx);
```

Comment:
once we have a `cmd` you should use `c->cmd->fullname` (not argv[0]`), it'll include the sub-command.
also a recent cleanup added `'` around the command name. 
e.g. 
```c
rejectCommandFormat(c,"wrong number of arguments for '%s' command", c->cmd->fullname);
```

Reply:
Fixed